android/libraries/rib-badoo/src/main/java/com/badoo/common/rib/RouterConfigurationManager.kt

```diff
@@ -0,0 +1,93 @@
+package com.badoo.common.rib
+
+import android.os.Parcelable
+import com.badoo.common.rib.RouterConfigurationManager.Effect
+import com.badoo.common.rib.RouterConfigurationManager.State
+import com.badoo.common.rib.RouterConfigurationManager.Wish
+import com.badoo.common.rib.RouterConfigurationManager.Wish.Pop
+import com.badoo.common.rib.RouterConfigurationManager.Wish.Push
+import com.badoo.common.rib.RouterConfigurationManager.Wish.NewRoot
+import com.badoo.common.rib.RouterConfigurationManager.Wish.Replace
+import com.badoo.mvicore.element.Actor
+import com.badoo.mvicore.element.Reducer
+import com.badoo.mvicore.element.TimeCapsule
+import com.badoo.mvicore.feature.ActorReducerFeature
+import io.reactivex.Observable
+import io.reactivex.Observable.empty
+import io.reactivex.Observable.just
+import kotlinx.android.parcel.Parcelize
+
+class RouterConfigurationManager<C : Parcelable>(
```

Comment:
Do you still need it?
It seems that only backstack manager used in the code above

Reply:
Yes, accidental left-over, will delete.